tox.ini

```diff
@@ -15,10 +15,13 @@ commands =
     pytest: pipenv sync --dev
     pytest: pytest
     format: black .
-    format: isort -m 3 --tc .
```

Comment:
What are these options that we're dropping?

Reply:
The `-m 3` is `multi_line_output = 3` and `--tc` is `include_trailing_comma = True`

Reply:
They are both included in the profile, which we were trying to reproduce earlier but was incomplete

Reply:
Ah, and black probably already incorporates these, yeah?